### ISSUE TITLE
Tooltips are single letters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#109]
```
What steps will reproduce the problem?
1. Modify a file
2. Hover the mouse over the "M" in the column

What is the expected output? What do you see instead?
The tooltip says "M".  I would expect "Modified" or something more detailed.  
Similar tooltips display for other letters (At least C for a conflict)

What version of the product are you using? On what operating system?
SvnX 1.2
MacOS 10.6
svn 1.6.12

Please provide any additional information below.

Pasted from SvnX changelog:
# Added a tooltip that explains the meaning of the columns’ characters. 

```

Original issue reported on code.google.com by `vanc...@gmail.com` on 19 Aug 2010 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
That tool-tip is automatically generated by Cocoa if the text is wider than the 
column.
Unfortunately Apple (ITUW) have changed the criteria under which they appear in 
OSX 10.6.
[In other words: This never happens on systems prior to OSX 10.6.]
The real tool-tip should still appear below it.  (Yuck.)

I've now fixed this and it will be in the next release.
```

Original comment by `chris...@gmail.com` on 25 Aug 2010 at 9:49
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3b1.
```

Original comment by `chris...@gmail.com` on 16 Sep 2010 at 12:35
- Changed state: **Fixed**
